app/jobs/icpsr_etl_job.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Job to ETL dataset metadata from ICPSR via Searchworks
+class IcpsrEtlJob < EtlJob
+  include Checkinable
+
+  # Search criteria for ICPSR datasets in Searchworks. We look for ICPSR listed
+  # as an author, and filter for datasets that have an online access URL.
+  def self.solr_params
+    {
+      q: 'Inter-university Consortium for Political and Social Research.',
```

Comment:
you could also make this an author_other_facet and set the a filter query for that as well, but this also works

Reply:
It seems there are a bunch of ways to slice this – you can also do `ICPSR (Series)` in the series_facet. When I checked, you get the exact same number of results (10,301) either way. I guess the cataloging is just really consistent 🤷🏻